StreamAPI/src/Stream.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Stream {
+
+    //Есть два класса
+    // Нужно написать функцию, которая по списку persons вернет список адресов тех
+    // людей, чей возраст больше 17 лет.
+
+    public class Person {
+        String name;
+        int Age;
+        Address address;
+    }
+    public class Address {
+        String street;
+        int houseNumber;
+    }
+
+    List<Person> persons = Arrays.asList();
+
+    Map <Person,Address> res = persons.stream()
```

Comment:
this code is outside of any method. Why this is so? 

The persons list is empty at this point and contains no objects. What do you want to achieve here?